DOCS/man/options.rst

```diff
@@ -4786,6 +4786,22 @@ OSD
 ``--osd-fonts-dir=<path>``
     See ``--sub-fonts-dir`` for details.  Defaults to ``~~/fonts``.
 
+``--osd-glyph-limit=<value>``
+    Set the maximum number of cached glyphs in libass cache. 0 means libass uses
+    its default value.
+
+    Default: 0.
+
+``--osd-bitmap-max-size=<value>``
```

Comment:
Shouldn't we also have this for `sub`?

Reply:
Done, also added osd-shaper and default to simple.